tests/integration/query/simple/with_filter/with_not_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
todo: Because we mentioned this on our call, here is the `2022` :laughing:

Reply:
Thanks for catching my sloppy copy pasting ;)

Reply:
✅